guides/nuxtjs-starter-guide/visual-editing.mdx

```diff
@@ -0,0 +1,26 @@
+---
+_schema: guide-item
+_uuid: "c711011c-121f-4bdd-abdd-0ce99b6ee15d"
+_created_at: "2023-06-22 06:54:28 +0000"
+title: "Visual editing"
+nav_title: "Visual editing"
+order: 4
+published: true
+image: "/documentation/static/CloudCannonDocumentationog.jpg"
+description: "This guide will walk through the steps required to get your Next.js site built, editable and live on CloudCannon."
+tags: []
+related_articles: null
+related_links: null
+---
+
+Visual editing on CloudCannon can be broken down into two separate parts:
+
+1. [Previews](/documentation/articles/live-editing-with-vue/)\: When data is changed in the Data Editor, the new data is sent to the page props.
+This allows the editor to see the changes as they make them.
```

Comment:
Maybe `This allows you to see the changes as you make them.`? Just because of the ambiguity between "Editor" (the CMS) and "editor" (the user of the CMS) 🤷